examples/feature_convert/build123d/cube.py

```
@@ -0,0 +1,7 @@
+import build123d as bd
+
+with bd.BuildPart() as result:
+    bd.Box(5, 5, 5)
+
+if "show_object" in locals():
+    show_object(result.part.wrapped, name="art")
```

Ruff (0.8.2)

Undefined name `show_object`

(F821)